Plugins/lib/windows/arm64/gilzoide-sqlite-net.dll.meta

```diff
@@ -48,25 +50,44 @@ PluginImporter:
     second:
       enabled: 1
       settings:
-        CPU: AnyCPU
+        CPU: None
   - first:
       Standalone: OSXUniversal
     second:
       enabled: 1
       settings:
-        CPU: AnyCPU
+        CPU: None
   - first:
       Standalone: Win
     second:
-      enabled: 0
+      enabled: 1
```

Comment:
Same as before, "Win" being the x86 version of Windows, this should probably be disabled.